Digital Shadows/digital_shadows_searchlight/_meta/fields.yml

```diff
@@ -0,0 +1,44 @@
+event.id:
+ name: event.id
+ type: keyword
+ description: Event UUID in Digital Shadows SearchLight
+
+event.reason:
```

Comment:
Every fields declared in this file must be considered as new fields brought by this intake. However, the `event.reason` already exists in ECS. You can remove its declaration from this file.